frontend/src/styles/LoginRegistrationPage.css

```diff
@@ -1,97 +1,246 @@
+  font-weight: var(--FontWeight);
+  height: 100%;
+  line-height: var(--LineHeight);
+  margin: 0;
+  min-height: 60px;
+}
+form{
+  padding-top: 60px;
+}
 div.login1-button {
+  text-align: center;
+  opacity: 0.7;
+  cursor: pointer;
+  font-family: var(--FontFamily);
   width: 45px;
+  font-size: 34px;
+  letter-spacing: 2px;
   float: none;
   margin: 0 auto;
-  margin-top: 5px;
+  margin-top: 10px;
   padding: 5px;
-  color: #1DE28F;
+  padding-top: 20px;
+  color:var(--themeColor);
   font-weight: bold;
-  cursor: pointer;
 }
-
-div.login2-button {
+div.login1-button:hover {
+  opacity: 1;
+}
+div.signup-button {
+   opacity: 0.8;
+   text-align: center;
     width: 175px;
+    font-family: var(--FontFamily);
+    font-size: 34px;
+    letter-spacing: 2px;
     float: none;
     margin: 0 auto;
     margin-top: 20px;
     padding: 5px;
-    color: #1DE28F;
+    color:#FFFFFF;
     font-weight: bold;
     cursor: pointer;
   }
-
+  div.signup-button:hover {
+      opacity: 1;
+  }
+  div.login2-button {
+     opacity: 0.8;
+     text-align: center;
+      width: 175px;
+      font-family: var(--FontFamily);
+      font-size: 32px;
+      letter-spacing: 2px;
+      float: none;
+      margin: 0 auto;
+      margin-top: 0px;
+      padding: 5px;
+      color: var(--themeColor);
+      font-weight: bold;
+      cursor: pointer;
+    }
+  div.login2-button:hover {
+      opacity: 1;
+    }
   div.registration-button {
     width: 200px;
     float: none;
     margin: 0 auto;
     margin-top: 20px;
     padding: 5px;
-    color: #1DE28F;
+    color:var(--themeColor);
     font-weight: bold;
     cursor: pointer;
   }
-
   div.registration2-button {
     width: 65px;
     float: none;
     margin: 0 auto;
     margin-top: 5px;
     padding: 5px;
-    color: #1DE28F;
+    color: var(--themeColor);
     font-weight: bold;
     cursor: pointer;
   }
-
-  .login-box-container {
-    text-align: center;
-    margin-left: auto;
-    margin-right: auto;
-    margin-top: auto;
-    width: 50%;
-    margin-left: auto;
-    display: block;
-    padding: 10px;
-    align-content: center;
-}
-
-.text-line {
-  background-color: transparent;
-  text-align: center;
-  color: #eeeeee;
-  outline: none;
-  outline-style: none;
-  border-top: none;
-  border-left: none;
-  border-right: none;
-  border-bottom: #1DE28F 2px solid;
-  width: 185px;
-  padding: 3px 10px;
-}
-
 div.logo img {
   text-align: center;
   width: 420px;
-  margin-bottom: 15px;
+  padding-top: 40px;
+  margin-bottom: 0px;
 }
-
 div.logo {
   text-align: center;
   margin-top: 0px;
 }
-
 .login-loader-holder {
   text-align: center;
   width: 80px;
   float: none;
   margin: 0 auto;
   height: 50px;
 }
-
 .login-loader {
   margin:auto;
 }
+.align {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: var(--gridMaxWidth);
+  width: var(--gridWidth);
+}
+input:before{
```

Comment:
use class names, don't define for "input" in general. This can have bad consequences.